pkg/volume/util.go

```diff
@@ -410,3 +410,27 @@ func JoinMountOptions(userOptions []string, systemOptions []string) []string {
 	}
 	return allMountOptions.UnsortedList()
 }
+
+// ZonesToSet converts a string containing a comma separated list of zones to set
+func ZonesToSet(zonesString string) (sets.String, error) {
+	zonesSlice := strings.Split(zonesString, ",")
+	zonesSet := make(sets.String)
+	for _, zone := range zonesSlice {
+		trimmedZone := strings.TrimSpace(zone)
```

Comment:
If we defined it as a space separated string, we could just use strings.Fields() and maybe not need a helper function at all?